dotcom-rendering/src/web/browser/initDiscussion/init.ts

```diff
@@ -20,7 +20,7 @@ function forceHydration() {
 		props.expanded = true;
 
 		// Force hydration
-		doHydration(name, props, guElement);
+		void doHydration(name, props, guElement);
```

Comment:
This relates to [this change](https://github.com/guardian/dotcom-rendering/pull/6170) I believe

Reply:
Yes, we are now able to use the `void` keyword, and it is recommended for promises that are not awaited.

Reply:
I quite like it!